nimterop/getters.nim

```diff
@@ -546,30 +546,30 @@ proc getPreprocessor*(gState: State, fullpath: string): string =
 
   # Include content only from file
   for line in execAction(cmd).output.splitLines():
-    if line.strip() != "":
-      if line.len > 1 and line[0 .. 1] == "# ":
-        start = false
+    # We want to keep blank lines here for comment processing
```

Comment:
This whole block is just a deletion of `if line.strip() != "":` and then re-indenting. The reason I got rid of this is because spacing provides hints on what comment goes with what piece of code.